src/api/app/views/event_mailer/create_report.html.haml

```diff
@@ -0,0 +1,33 @@
+:ruby
+  def link_to_reportables
+    reportable = Report.find(event['id']).reportable
+    case event['reportable_type']
+    when 'Comment'
+      link_for_commentables_on_reportables(commentable: reportable.commentable)
+    when 'Package'
+      package_show_path(package: reportable, project: reportable.project, anchor: 'comments-list')
+    when 'Project'
+      project_show_path(reportable, anchor: 'comments-list')
+    when 'User'
+      user_show_path(reportable)
+    end
+  end
+
+  def link_to_commentables_on_reportables(commentable:)
+    case commentable
+    when BsRequest
+      request_show_path(commentable.number, anchor: 'comments-list')
+    when BsRequestAction
+      request_show_path(number: commentable.bs_request.number, request_action_id: commentable.id, anchor: 'tab-pane-changes')
+    when Package
+      package_show_path(package: commentable, project: commentable.project, anchor: 'comments-list')
+    when Project
+      project_show_path(commentable, anchor: 'comments-list')
+    end
+  end
+
+%p
+  = User.find(event['user_id']).login
+  reported a #{event['reportable_type'].downcase}
```

Comment:
You are missing `Report.reason`

Reply:
done

Reply:
will also add it to the web notifications